pkg/generators/golang/errors_generator.go

```diff
@@ -427,6 +454,9 @@ func (g *ErrorsGenerator) generateCommonErrors() error {
 				case "operation_id":
 					object.operationID = iterator.ReadString()
 					object.bitmap_ |= 16
+				case "details":
+					object.details = iterator.ReadAny()
```

Comment:
@jhernand I am not sure if this is the correct function to call on an `interface{}` seems the best fit, as they dont have an explicit `ReadInterface()` func

Reply:
I think it is the right function. But that function returns an "Any" type. You will have to call "GetInterface" on that in order to convert it to the "interface{}" type used by "object.details".